docs-website/i18n/ar/docusaurus-plugin-content-docs/current/guides/recitations-ayah-timings.md

```
@@ -0,0 +1,148 @@
+sidebar_position: 5
+title: التلاوات وتوقيتات الآيات
+description: كيفية جلب مسارات صوت التلاوة ومزامنة توقيتات الآيات لتشغيل الوسائط.
+---
+
+# التلاوات وتوقيتات الآيات
+
+زامن الصوت مع النص على مستوى الآية.
+
+## نموذج البيانات
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)